sass/layout/_no-sidebar.scss

```diff
@@ -1,5 +1,15 @@
 .no-sidebar {
 
+	.site {
+		display: grid;
+		grid-template-columns: auto;
+		grid-template-areas:
+			"header"
+			"main"
+			"sidebar"
+			"footer";
+	}
+	
```

Comment:
It seems that there is an unexpected whitespace here that must be removed. That's why travis is failing.

Reply:
Done